src/github/pullRequestManager.ts

```diff
@@ -1033,7 +1041,7 @@ export class PullRequestManager implements vscode.Disposable {
 		return;
 	}
 
-	async validateDraftMode(pullRequest: PullRequestModel): Promise<boolean> {
+	async validateDraftMode(pullRequest: IssueModel): Promise<boolean> {

```

Comment:
I don't think it's possible to have draft issue, so let's keep this as `PullRequestModel` here on on `updateDraftModeContext`

Reply:
There is no calling code that would pass in an IssueModel instead of a PullRequestModel here, but there's nothing stopping us from doing that some day. Since it dosn't need to be a PullRequestModel, can we leave it as an IssueModel?